BEIMA.Backend.FT/TestObjects.cs

```diff
@@ -8,7 +8,7 @@ public class TestObjects
     {
         public class Device
         {
-            [JsonProperty(PropertyName = "_id")]
+            [JsonProperty(PropertyName = "id")]
```

Comment:
I think this might cause some of our front end testing to fail, but I could be wrong.

Reply:
Do the frontend tests call the ft tests?

Reply:
I'm wrong, I didn't notice that since I changed the device model to serialize back into a c# object it would return id and not _id.